api/v1/tetragon/events.proto

```diff
@@ -84,10 +84,12 @@ message CapFilterSet {
 }
 
 message RedactionFilter {
-    // Match events that the redaction filter will apply to.
-    repeated Filter match = 1;
+    // Deprecated, do not use.
+    repeated Filter match = 1 [deprecated=true];
```

Comment:
by the way I think it makes sense to have both the specific optimized redactions for binary_regex and also the general ones even if they are slower. wdyt? (note not requesting a change for this PR but maybe something to consider.)

Reply:
I think we can definitely consider this moving forward.